x-pack/plugins/reporting/public/management/components/report_info_flyout.tsx

```diff
@@ -139,7 +141,7 @@ export const ReportInfoFlyout: FunctionComponent<Props> = ({ config, onClose, jo
           {isLoading ? (
             <EuiLoadingSpinner />
           ) : loadingError ? undefined : !!info ? (
-            <ReportInfoFlyoutContent info={info} />
+            <ReportInfoFlyoutContent info={info} isServerless={isServerless} />
```

Comment:
1. Let's change this to passing the `ClientConfigType` object 
```suggestion
            <ReportInfoFlyoutContent info={info} config={config} />
```